PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -283,3 +283,12 @@ def nanoL1TrigObjCustomize(process):
 def nanoL1TrigObjCustomizeFull(process):
     process.nanoTableTaskCommon.add(process.l1TablesTask)
     return process
+
+def addExtraEGammaVarsCustomize(process):
```

Comment:
Hi @swagata87,

can you please move the customize function in a separate cff (egamma_custom_cff.py)? This to be in line with the changes that will be introduced by: https://github.com/cms-sw/cmssw/pull/42238

Reply:
Done,
with this change, the customise command in cmsDriver would be `--customise PhysicsTools/NanoAOD/egamma_custom_cff.addExtraEGammaVarsCustomize`